docs/tutorials/02_create_your_first_app/05_using-collections.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 5
+title: Step 5 - Using collections
+---
+
+# Step 5 - Using collections
+
+In this step we will add new variable `options` to the actor that will hold a list of the choices that users can vote for in the poll.
+
+
+## Creating a "map"
+
+Each option is a string, what describes what the users are voting for. However, since we have multiple options in the poll we will need to store them in some data structure. We will use a collection called [`RBTree`](/motoko/main/base/RBTree.md) which is similar to "maps" or "dictionaries" in other languages.
+
+The collection will "map" option identifier (a string of type `Text`) to the option value (a natural number of type `Nat`). For example, we may have this list of options if we want to vote for our favorite programming language:
+
+| id  | Vote Count |
+| --- | ------ |
+| "Motoko"   | 0 |
+| "Rust"   |   0 |
+| "TypeScript"   |  0 |
+| "Python"   | 0     |
+
+
+To use RBTree we need to import it. In addition, we will need to import the standard type [`Nat`](/motoko/main/base/Nat.md) to use some of the module's functions. So first let's add an import statement to the beginning of our file `main.mo`:
+```motoko
+import RBTree "mo:base/RBTree";
+import Nat "mo:base/Nat";
+```
+
+Next we need to declare the variable of this type inside the actor:
+```motoko
+    var votes: RBTree.RBTree<Text, Nat> = RBTree.RBTree(Text.compare);
+```
+
+In this snippet we create a variable `votes`, whose type is `RBTree.RBTree<Text, Nat>`.
+
+:::note
+The current version of Motoko makes collections declarations a bit verbose and difficut to understand for beginners. We are working to create a more elegant syntax.
```

Comment:
This phrasing makes it feel like Motoko in its current state. I would phrase it more along the lines of "Future versions of Motoko will introduce usability enhancements in this area"